cylc/flow/subprocctx.py

```diff
@@ -158,3 +158,10 @@ def get_signature(self):
         args = self.func_args + [
             "%s=%s" % (i, self.func_kwargs[i]) for i in skeys]
         return "%s(%s)" % (self.func_name, ", ".join([str(a) for a in args]))
+
+    def __str__(self) -> str:
+        """
+        >>> str(SubFuncContext('label', 'func', [1, 2], {'a': 3}))
+        '[xtrigger-func cmd]: func(1, 2, a=3)'
+        """
+        return f"{SubProcContext.__str__(self)}: {self.get_signature()}"
```

Comment:
Do we need the `[xtrigger-func cmd]` prefix?
```suggestion
        return self.get_signature()
```

Reply:
Dunno 😄